### PR TITLE
docker-app: Fix clang-tidy warnings

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
I'm surprised CI hasn't caught this. It looks like the .travis.yml has `-eTEST_WITH_DOCKERAPP=1` when it calls test.sh

File: src/libaktualizr/package_manager/dockerappmanager.cc [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 
 struct DockerApp {
-  DockerApp(const std::string app_name, const PackageConfig &config)
-      : name(std::move(app_name)),
-        app_root(std::move(config.docker_apps_root / app_name)),
-        app_params(std::move(config.docker_app_params)),
-        app_bin(std::move(config.docker_app_bin)),
-        compose_bin(std::move(config.docker_compose_bin)) {}
+  DockerApp(std::string app_name, const PackageConfig &config)
+      : name(app_name),
```
Comment (patrickvacek) [inline src/libaktualizr/package_manager/dockerappmanager.cc]: I'm also a bit surprised. It all looks fine except for this bit: why not move `app_name`? Seems like an unnecessary copy.

Comment (doanac) [reply src/libaktualizr/package_manager/dockerappmanager.cc]: Great question. I lost about 30 minutes of debugging time on that very issue. The problem is this line of code that comes *after* that std::move: `app_root(config.docker_apps_root / app_name),`. The app_name passed to that becomes an empty string and `app_root` winds up not being correct.

**Comment by patrickvacek** [comment]
Also, if you rebase on latest master, I think we can fix the Travis timeout problem.

File: src/libaktualizr/package_manager/dockerappmanager.cc (hunk 1 above)
Comment (patrickvacek) [reply src/libaktualizr/package_manager/dockerappmanager.cc]: Oh wow, I see. Can you not use `name` instead of `app_name` in the latter line?

Comment (doanac) [reply src/libaktualizr/package_manager/dockerappmanager.cc]: ah - better idea. i've done a force-push with rebase on master + plus this suggestion.

**Comment by lbonn** [comment]
The reason it was not caught by CI is that `TEST_WITH_STATICTESTS` and `TEST_WITH_DOCKERAPP` are not enabled simultaneously on CI right now.
We could add `TEST_WITH_DOCKERAPP` to the debian testing jobs on Travis and gitlab.

**Pull request merged**